docs/markdown/snippets/gnome.md

```diff
@@ -0,0 +1,13 @@
+## `gnome.compile_schemas()` sets `GSETTINGS_SCHEMA_DIR` into devenv
+
+When using `gnome.compile_schemas()` the location of the compiled schema is
+added to `GSETTINGS_SCHEMA_DIR` environment variable when using
+[`meson devenv`](Commands.md#devenv) command.
+
+## `update_desktop_database` added to `gnome.post_install()`
+
+Applications that install a `.desktop` file need to update the cache upon
```

Comment:
Applications that install a `.desktop` file **containing a `MimeType`**.

Reply:
done.